webapp/src/main/webapp/.gitignore

```diff
@@ -1,9 +1,4 @@
-/errorhandler.jsp
-/memoryUsage.jsp
-/openmrs_green.css
-/openmrs_legacy.css
-/openmrs_orange.css
-/openmrs_purple.css
-/openmrs.css
-/openmrs.js
-/style.css
+*.jsp
```

Comment:
I think you should ignore everything except for the files in this folder, because modules can have other file types like htm and html files that get copied here